end-to-end-test/shared/specUtils.js

```diff
@@ -699,6 +699,42 @@ function selectClinicalTabPlotType(type) {
     ).click();
 }
 
+function clickColumnSelectionButton(patientTable) {
+    $(`[data-test=${patientTable}]`)
+        .$('button*=Columns')
+        .click();
+}
+
+function selectColumn(namespaceColumn1) {
+    $(`[data-id="${namespaceColumn1}"]`).click();
+}
+
+waitForMutationTable = () => {
```

Comment:
This is not specific enough since there could possibly multiple LazyMobxTables.  A developer would assume that this would actually idenify and wait for a mutation table but it doesn't.     ```$(".resultsPageMutationsGeneTabs .lazy-mobx-table")``` in the body of your test would be fine.  We should avoid making helpers and putting them in shared specUtils unless they really provide value.

Reply:
Fixed in [45e5ee3](https://github.com/cBioPortal/cbioportal-frontend/pull/4424/commits/45e5ee3ba491ee838276c3807739a92488f62ed1)